pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/MinusAllOperator.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.query.runtime.operator;
+
+import java.util.List;
+import javax.annotation.Nullable;
+import org.apache.pinot.common.utils.DataSchema;
+import org.apache.pinot.core.data.table.Record;
+import org.apache.pinot.query.runtime.plan.OpChainExecutionContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * EXCEPT ALL operator.
+ */
+public class MinusAllOperator extends SetOperator {
+  private static final Logger LOGGER = LoggerFactory.getLogger(MinusAllOperator.class);
+  private static final String EXPLAIN_NAME = "MINUS ALL";
```

Comment:
@gortiz I just noticed that other operators don't use spaces in their explain name. Should this be `MINUS_ALL` or `EXCEPT_ALL`? Also, it doesn't seem like this is even used for the multi-stage query engine's explain plan output? If it's only for the v1 query engine then I guess it isn't too relevant anyway since these set operators can only be used in the intermediate stages of the v2 query engine right?

Reply:
Hm actually, I guess we don't really need separate operators for these. We could instead just pass a boolean flag indicating whether or not the `all` modifier is set to the existing operators.

Reply:
I think this is only used when executing `explain implementation plan for`. Can you verify if the flag is seen there?

Reply:
The physical explain plan gets the explain name for the operator from the `SetOpNode` [here](https://github.com/apache/pinot/blob/0f48825d2d36e489c743d7e50bdb62bfc1f786ba/pinot-query-planner/src/main/java/org/apache/pinot/query/planner/plannode/SetOpNode.java#L60) (called from [here](https://github.com/apache/pinot/blob/0f48825d2d36e489c743d7e50bdb62bfc1f786ba/pinot-query-planner/src/main/java/org/apache/pinot/query/planner/explain/PhysicalExplainPlanVisitor.java#L120)). Do you think it makes sense to modify `SetOpNode::explain` to include whether or not the `ALL` modifier is set (something like `INTERSECT_ALL` / `MINUS_ALL`)?